plans/tier1.fmf

```diff
@@ -1,372 +1,372 @@
-/activation_key:
-  discover+:
-    test: checks-after-conversion
-  prepare+:
-  - name: main conversion preparation
-    how: shell
-    script: pytest -svv tests/integration/tier1/method/activation_key.py
-  - name: reboot after conversion
-    how: ansible
-    playbook: tests/ansible_collections/roles/reboot/main.yml
+activation_key:
```

Comment:
The `/` here is not needed?

Reply:
Oh, good catch!
It is indeed, linter did not complain, I didn't notice somehow.
Thank you @r0x0d !